src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsitetruemoney.phtml

```diff
@@ -0,0 +1,18 @@
+<?php $_code = $this->getMethodCode(); ?>
+
+<ul id="payment_form_<?php echo $_code; ?>" class="form-list" style="display:none;">
+    <!-- True money phone number form-->
+    <li>
+        <label for="<?php echo $_code; ?>_phone_number" class="required">
+            <em>*</em><?php echo $this->__('Phone Number'); ?>
+        </label>
+        <div class="input-box">
+            <input id="<?php echo $_code; ?>_phone_number" type="text"
```

Comment:
If the user is logged in and has a phone number in their default address, we should probably automatically have it filled in to the phone number field. I believe Jacek did something similar on Magento 2

Reply:
@jonrandy it's done.